### ISSUE TITLE
required version of typed-ast seems obsolete

### ISSUE THREAD

**Issue by brauliobarahona** [abulka/pynsource#67]
When installing in MacOS

```
ERROR: astroid 2.2.5 has requirement typed-ast>=1.3.0; implementation_name == "cpython", but you'll have typed-ast 1.1.1 which is incompatible.
ERROR: black 19.10b0 has requirement typed-ast>=1.4.0, but you'll have typed-ast 1.1.1 which is incompatible.
```
updating the requirements file to `typed-ast 1.4.0`seems to solve the installation problem.

**Comment by abulka** [comment]
Pynsource is not currently compatible with the latest `typed-ast` 1.4.0.

There were some breaking changes made to typed-ast when they moved to 1.4.0 and I haven't yet modified Pynsource to take these into account.  For now, stick with `typed-ast 1.1.1` as per `requirements.txt`.

As for that astroid error in your message, astroid is not listed in requirements.txt. 

P.S. Actually I too happen to have astroid installed on my development system (at vers 2.3.3) - I must have done that at some stage for some reason, then downgraded `typed-ast`back to 1.1.1 to get Pynsource to work again.

**Comment by abulka** [comment]
Now compatible with latest typed_ast
47250861b55d262bdea0d043814a6751f4af702f

Note, even though `typed_ast` has been merged into regular Python ast as of Python 3.8, we still need `typed_ast` because it supports parsing Python 2, whereas Python 3's ast does not.
